week-3/learning-outcomes.md

```diff
@@ -0,0 +1,26 @@
+# Week 3 Learning Outcomes
+
+## Testing
+
+- [ ] **We can test the functionality of our app**
+- [ ] We can write tests to mimic the behaviour of a user performing different actions
+- [ ] We can check that passing a given input into our tests returns the expected output
+
+### Modular functions
+
+- [ ] **We can write testable, modular functions**
+
+### HTML forms
+
+- [ ] **We can submit an HTML form without using JavaScript**
+
+### Scope
+
+- [ ] **We can use function and block scope to store values inside temporary variables**
```

Comment:
I think this might be confusing for them. I don't know if "temporary" will make sense for any variables at this point.

Scope is mostly about isolation, so maybe something like:

```suggestion
- [ ] **We can isolate variables inside function and block scopes**
```